src/index.js

```diff
@@ -170,6 +174,7 @@ async function bundleMDX(
     return {
       code: `${code};return Component.default;`,
       frontmatter,
+      errors: bundled.errors,
```

Comment:
I added this in testing but I wonder if it should stay? makes sense to me to return the esbuild errors incase a user is using a try/catch block and wants to do a bit of inspection.